pages/wizard/rhv/storage.py

```diff
@@ -15,6 +15,21 @@ class Storage(Base):
     _hosted_domain_name_loc = (By.ID, "hosted_storage_name")
     _hosted_storage_address_loc = (By.ID, "hosted_storage_address")
     _hosted_share_path_loc = (By.ID, "hosted_storage_path")
+    _alert_rhv_storage = (By.XPATH, "//div[contains(@class, 'rhci-alert')]")
+    _spinner_storage_mount = (By.XPATH, "//div[contains(@class, 'spinner-md')]")
```

Comment:
It's a holdover from testing an invalid NFS mount.  If you use a bad address for the network share, you will see the spinner for a few minutes until the mount attempt times out.  The current rhv test deployment uses a valid address so  there isn't a huge need for it in the current test.  If there was an issue with fusor reaching the NFS host you would wait for the storage mount spinner to disappear before checking for an alert.